http/README.md

```diff
@@ -95,4 +101,9 @@ channels:
             bindingVersion: '0.2.0'
 ```
 
-[schemaObject]: https://www.asyncapi.com/docs/specifications/latest/#schemaObject
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
merge leftovers

Reply:
Being dealt with in https://github.com/asyncapi/bindings/pull/220 and https://github.com/asyncapi/bindings/pull/221